packages/@aws-cdk/aws-backup/README.md

```diff
@@ -132,8 +132,10 @@ const vault = new backup.BackupVault(this, 'Vault', {
 })
 ```
 
-Use the `blockRecoveryPointDeletion` property to add statements to the vault access policy that
-prevents recovery point deletions in your vault:
+Alternativately statements can be added to the vault policy using `addToAccessPolicy()`.
+
+Use the `blockRecoveryPointDeletion` property or `blockRecoveryPointDeletion()` method to add
```

Comment:
```suggestion
Use the `blockRecoveryPointDeletion` property or the `blockRecoveryPointDeletion()` method to add
```

Can you add an example in the readme for `blockRecoveryPointDeletion()` as well?